### ISSUE TITLE
Notes: Questions about their wording, formatting, linking

### ISSUE THREAD

**Issue by lizoknj** [rbms-bsc/DCRMR#44]
Background: We decided that every instruction within the main element should also be represented in the appropriate Note on/Details of element. This is a departure from DCRM(B) practice, which generally did not repeat in the Notes chapter instructions on notes that had appeared in the earlier chapters. It makes sense to have the instructions for notes appear in both places, but I’d like to clarify how to do this without sacrificing clarity, but also without undue repetition.

1.	How much of the text of the original instruction to preserve in the note version?

In cases when the original instruction to make a note is comprehensible on its own, I have just copied the text to the corresponding note element.

When the original instruction to make a note needs some context/background but the full instruction would be too wordy, I have rewritten the note to provide that context briefly,  e.g.:

REWRITE: “If the extent is not fully recorded in the extent of manifestation element, record all the sequences in a note on extent of manifestation if considered important” [omits all the verbiage on when and how to to record a shortened extent in the extent of manifestation element]

Is it okay to rewrite/shorten, or should the entire wodge of text from the full instruction be copied over? 

Also, I have retained in the note element instruction the name of the note element as it appeared in the original instruction (e.g. ,” record all the sequences in a note on extent of manifestation if considered important” instead of ”record all the sequences in a note if considered important”). Since every note within a given note element is going to be a note on that element, it seems redundant to include the full name of the note. What do you think?

2.	Examples, what to include, what to leave out

Omitting the transcribed/recorded statement from the example:

In DCRM(B) Chapters 1-6, instructions to make notes are almost always paired with the transcribed/recorded information that is the subject of the note,  each on a separate line. In Chapter 7, the note appears on its own, without the statement; occasionally the original statement is quoted in a separate “Comment:”, e.g. 7B6.5:
	By Gilbert Burnet, Bishop of Salisbury
        (Comment: Statement of responsibility reads: “by the Right Reverend Father in God, Gilbert Lord Bishop of Sarum”) 

Most of the examples in the PS’s for note elements follow the pattern in Chapter 7, but some present both a named element, usually a transcribed statement,  and a Note on [element] together on separate lines:
		[Name of element]: ….
		Note on [element]: ...

Is it ever necessary to include the statement in the example in the note element? There will always be a link back to the main element, and a comment can be used in cases when the transcribed statement is needed to make sense of the note, as in the DCRM(B) example quoted above.

Omitting labels in examples within note elements:
 
(Similar issue to including the full name of the note in the instruction) In examples of notes used in the main elements, we have been using labels that cite the note  element by its full name (as opposed to just “Note:”  or “Optional note”, as in DCRM(B)). This makes sense in the main elements, because in RDA, unlike AACR, so many notes are named elements as opposed to generic notes.  Is there any reason to do this in the note elements, though, since every example appearing in a given element will be an example of that particular note?

3.	Links from note element to main element:

Are we going to be linking from the note elements to the elements they correspond to (and vice versa)? At what level? From the note element as a whole to its corresponding element as a whole, i.e.  from “note on extent of manifestation” to “extent of manifestation"? Or from a specific instruction in the note element to a specific instruction in the corresponding element?  Is it possible to do the latter? It is more work, certainly, but would be more convenient for the user.  On the other hand, the user could always do a string search within the element to find the specific instruction, so either way is fine. There are also issues with work flow, since the markdown volunteers (as I understand it) will be the persons inserting the numbers. If the link is just to the element as a whole, it is easy enough for the markdown volunteer to locate the numbers for the main elements linked to. If the link is supposed to be to a specific instruction, it is more work for the volunteer to identify the number.

Also, how should the link be expressed in terms of visible text? If the link is at the element level, there could just be a link at the head of each element, I guess. But if the link is from a specific instruction, then how to word the link? 

                See custodial history of item
       OR: 	See custodial history of item, for summaries of the past ownership of an item (linking to specific instruction)
       OR	See 8.22
       OR	a combination:  See custodial history of item, 8.22
       OR    See 8.221 (If the target is a specific instruction)

Which is better?  And should we preface the link with “See”, or would it suffice to enclose the text for the link, in parentheses, using the color/font that indicates a hyperlink?  Relying on the color/font of the hyperlink has issues for the visually impaired, or for people who prefer to use a printed out version of the rules.
 
Any thoughts on this, or kindred issues, will be much appreciated.

**Comment by ladylazarus3** [comment]
Q. How much of the text of the original instruction to preserve in the note version?

The original google doc for [Note on publication statement](https://docs.google.com/document/d/1IPC7rc87zw-j4q3fb7WSY82wm4PA5WR4tPf7biD92hY/edit?usp=drive_web&ouid=116566503212939656580) was fairly short and taken from 7B8.  When we decided to repeat instructions for notes creating text for both in the specific entity page and in the "Note on [X}" entity page, i found it very helpful as it sort of gives a cataloger a quick "checklist" of all the notes types possible and their association with another element.

This is the formula i used to fill out the [Note on publication statement entity page](https://rbms-bsc.github.io/DCRMR/books/ppdm/Note-on-publication-statement/) based off the DCRM Editorial guidelines.

**The Note formula**

"Make a note on [name of note entity] if [condition that makes the note applicable in the situation derived from DCRM(B) text] (see [chapter#.entity#.section#.instruction#]).

Much of the "if" conditional statement directly referencing the particular type of note is copied from the corresponding element.  i've slightly rewritten where the full instruction is too wordy or includes instructions not relevant to the note instructions.  So in your rewrite example, the only addition i would have made using the formula above would be to add this first bold bit:

REWRITE: “**Make a [note on the Extent of manifestation](https://rbms-bsc.github.io/DCRMR/books/physical-description/Note-on-extent-of-manifestation/)** if the extent is not fully recorded in the extent of manifestation element, record all the sequences in a note on extent of manifestation if considered important” [omits all the verbiage on when and how to to record a shortened extent in the extent of manifestation element]

So, yes, it is ok to rewrite/shorten the wodge of text from the full instruction copied over and the very crude formula above is how i accomplished that for the PPDM chapter (the ppdm chapter is a guinea pig and i fully expect it will change more over time). 

including the full name of the specific note is a way to get folks familiar with the RDA Element set and makes the DCRMR instructions a pedagogical tool as well as a descriptive standard.  It is similar to how we use various 5XX MARC tags in daily discourse today (although we would never use MARC tags in DCRM instructions, even in the examples), getting folks used to using the RDA element set names is a helpful tool in learning the bits of RDA that have to be in DCRMR.

Q. Is it ever necessary to include the statement in the example in the note element? 

The statement includes important context for the note and will keep folks from flipping back and forth between various DCRMR pages/browser tabs needlessly.  Using the name of the element and the note on the element together conveys stronger meaning.  For example from [Note on publication statement 4.206.3.4](https://rbms-bsc.github.io/DCRMR/books/ppdm/Note-on-publication-statement/#4.206.3.4) i like:

> Example: 

> Publication statement: Sadopolis : Chez Justin Valcourt ... à l'enseigne de la Vertumalheureuse, an 0000 [i.e. Brussels : Jules Gay, 1866]

> Note on publication statement: Corrected imprint from: Pia, P. Livres de l'Enfer

much better than what is currently is included on the web page as it teaches folks the names of elements by them looking at the example itself.

Q. Are we going to be linking from the note elements to the elements they correspond to (and vice versa)? 
Yes, the instructions for notes and instructions to the corresponding element should link to the specific instruction in the note element to a specific instruction in the corresponding element.  You can see examples of this in [Note on publication-statement](https://rbms-bsc.github.io/DCRMR/books/ppdm/Note-on-publication-statement/#4.206.3.4) and its corresponding instruction in [Publication statement](https://rbms-bsc.github.io/DCRMR/books/ppdm/Publication-statement/#4.20.5.1).

Folks can always add a link to a particular instruction by using the four digit citation number at the end of the url preceded by the # symbol.

`https://rbms-bsc.github.io/DCRMR/books/ppdm/Publication-statement/`  links to the entity page.
`https://rbms-bsc.github.io/DCRMR/books/ppdm/Publication-statement/#4.20.5.1` links to an individual instruction on that entity page.  i agree that this is more convenient for the user.   Keepers are double-checking links and citation numbers as we process pull requests from volunteers.  Linking and locating the correct instruction does take a lot of time, Liz, if you'd rather us add the correct link and you just supply the link text that is fine.  We are going over pull requests with a fine tooth comb.

Q.  Also, how should the link be expressed in terms of visible text? If the link is at the element level, there could just be a link at the head of each element, I guess. But if the link is from a specific instruction, then how to word the link?

i used the "See 8.22" method in my [Note on publication statement](https://rbms-bsc.github.io/DCRMR/books/ppdm/Note-on-publication-statement/#4.206.3.4) instructions, but i actually really really link the combination as that seems to both teach/reinforce the names of the RDA elements as well as adding a user-friendly link between the instructions.

In essence, if we follow the combination method the text of [4.206.3.4](https://rbms-bsc.github.io/DCRMR/books/ppdm/Note-on-publication-statement/#4.206.3.4) the revised instruction would read:

> 4.206.3.4 Make an explanatory note if all information relating to the publication statement appearing in the manifestation is known to be fictitious or incorrect (see Publication statement, 4.20.5.1).

i have a strong preference for the combination citation method for all the reasons Liz points out.

**Comment by lizoknj** [comment]
I'm still digesting Kalan's very helpful comments about notes. I like the idea of standardizing the wording to begin with "Make a note".  Also helpful to know that the instruction in the main text to make a note will link to the specific instruction in the notes elements (and vice versa).
Re numbering: I think it makes sense to have the markdown volunteers be responsible for inserting the numbering (Kalan and Amy will backstop our efforts). I am still studying the principles of assigning the new decimal numbers. I'm sure I will have questions, but I will hold them until I have learned as much as I can.

**Comment by ladylazarus3** [comment]
re: numbering.

Yes, numbering is tricky... and there is a lot of grey.  i've been basically studying the DCRM(B) for its sleek and wonderful organization :) and defaulting to that in times of question.  i've also started a [spreadsheet for Xref between DCRM(B) and DCRMR](https://docs.google.com/spreadsheets/d/1nP4uJPxj6gbfWPcPusKxAqxBXbQW9HG8uoU1SxZMqq0/edit#gid=2022670504) and have been annotating my paper DCRM(B) with the new citation numbering just as a double-check to make sure we didn't accidentally skip anything.  Catalogers LOVE LOVE LOVE crosswalks (at least, this one does :) ).

The numbering for most chapters can be broken down like this:
chapter#.entity#.section#.instruction#

The chapter number and entity number are always available from the [order of the text spreadsheet](https://docs.google.com/spreadsheets/d/1nP4uJPxj6gbfWPcPusKxAqxBXbQW9HG8uoU1SxZMqq0/edit#gid=0).  The section number will always be formatted as a header in the Markdown file (the line of Markdown starts with two hashtags "##" and is formatted in the Markdown as a header (e.g. Element information, RDA Definition and scope, General rule, etc.).   For example:

`## Section numbering.  This line is formatted as a header` becomes:
## Section numbering. This line is formatted as a header

The individual instruction numbers fall under each section (i.e. the lines formatted as a header) and individual instruction numbering restarts at the number one under each section (i.e. the lines formatted as a header).

One thing we need to keep constantly in the back of our minds, i think, is that the DCRMR text **will not have page numbers**.  In this web format, the citation numbers become an invaluable tool in helping catalogers successfully navigate the DCRMR text.  This is why i am a fan of using citations in areas where there *may have not been citations previously* (preface and individual paragraphs of text in the introduction of DCRM(B) ).  i know that is controversial, but i really do think we will thank ourselves in the long run.

In DCRM(B), one could have said, "i looked at the 'notes' paragraph on the bottom of page 19 of the PDF" to guide my decision on phrasing my notes in the imperative form."

In DCRMR, one would have to say, "i looked at the [Examples and notes](https://rbms-bsc.github.io/DCRMR/books/introduction/Examples-and-notes/) web page under [I.2.1](https://rbms-bsc.github.io/DCRMR/books/introduction/Examples-and-notes/#I.2.1) to guide my decision on phrasing my notes in the imperative form."

The sections (i.e. the lines formatted in Markdown as headers) have built-in permalinks that are really long and rather inconsistent.  For example, the permalink to a section header is:
`https://rbms-bsc.github.io/DCRMR/books/introduction/Examples-and-notes/#i2-notes`
but the permalink to an individual instruction is:
`https://rbms-bsc.github.io/DCRMR/books/introduction/Examples-and-notes/#I.2.1` so folks don't need to know the name of the section in order to link or navigate to it-- you just need to know the citation.  i think we should encourage folks early and get them consistently in the habit of citing instruction numbers.  It isn't to say that folks can't use the section header permalinks.  It is just that those permalinks are constructed a bit different-- and there is absolutely nothing we can do to change how they are constructed.   It is a limitation of the system, but not one that should stop us, just one to be aware of.   On the other hand, if folks know the full citation number and know the name of the chapter and name of the entity, they can navigate straight to it.  Some of the section names get really long (e.g. [supplied and conjectured names of publishers, distributors, etc.](https://rbms-bsc.github.io/DCRMR/books/ppdm/Name-of-publisher/#42039-supplied-and-conjectured-names-of-publishers) is much longer than [4.203.9.1](https://rbms-bsc.github.io/DCRMR/books/ppdm/Name-of-publisher/#4.203.9.1).  It might be unlikely that folks will verbally reference or memorize citations at all, but it does help manage user expectations when navigating the text and makes our lives easier because we don't have to type out such very long URLs. :)

kalan

**Comment by lizoknj** [comment]
Thanks for this extra guidance. Great minds think alike, I created my own
table for physical description containing all the DCRM(B) instruction
numbers, heading names, and a snatch of the instructions, so I could peg
the new DCRMR numbers to the old. For the most part, they march along
together very well, there will be issues in some places (I started with
extent, which is probably the longest element in the whole chapter).

Liz

**Comment by ladylazarus3** [comment]
Excellent!  feel free to add to the [xreference spreadsheet](https://docs.google.com/spreadsheets/d/1nP4uJPxj6gbfWPcPusKxAqxBXbQW9HG8uoU1SxZMqq0/edit#gid=2022670504), if you'd like, but don't feel obligated to do so.  i have as a step in the drafty draft instructions on [how to process pull requests as a Keeper of the Text instructions](https://github.com/rbms-bsc/DCRMR/wiki/How-to-Process-Pull-Requests-as-a-Keeper-of-the-Text) in the [DCRMR Editorial Wiki](https://github.com/rbms-bsc/DCRMR/wiki).  Hm.  i should probably finish those instructions... :)  *cough* they are more drafty drafty than usual. :)  i kinda see this as one of those "polish" steps after the pull requests are merged to the master... at least... that made the most sense in my head.  *blushes*

kalan

**Comment by mjmascaro** [comment]
I admit I found the formula "Make a note on [name of note entity] if [condition that makes the note applicable in the situation derived from DCRM(B) text" did not work well for several notes on edition statement.  In that section, in many cases there was specific instructions on what to record in a note if a particular situation was met.  Trying to explain what to record in the note before explaining in what situation it was applicable to I thought was too ackward so I adjusted for readability in many case.  Totally to open further wordsmithing and refinement of construction in that section for more consistency and wanted to present  this section as a case study of some of the complexity of certain notes for further discussion.

**Comment by elizhobart** [comment]
@mjmascaro I just reviewed the Note on edition statement section. Your rewriting looks makes sense to me. Were there other places in the edition chapter where this came up?

**Comment by lizoknj** [comment]
I'm seeing two patterns (there are probably more) for structuring note elements. 

One is to provide a heading for each instruction that describes the situation (see Note on edition statement.md). no general heading, such as  Notes on Edition statement or whatever, is provided.  

The other approach (in, e.g. Note on publication statement)  is to provide a more general heading, e.g. Notes from Publication statements, or Notes from Name of publisher, and then just put the notes under that heading without individual subheadings (each note is given a distinctive final number). 

I can see advantages to both approaches. Do we want to choose a single method, or let the nature of the element dictate the pattern? 

Liz O'Keefe

**Comment by mjmascaro** [comment]
@elizhobart thanks for reviewing.  There were no other places in the edition chapter this came up.

**Comment by mjmascaro** [comment]
To follow up on @lizoknj question.   As the text finalizer of note on editions I wanted to provide some additional background on my choices.  All the unique notes in the edition chapter appear in edition statement (or the chapter page), so it did not make sense to provide subheadings for each element in this case.  (Note references in other edition elements are repeated instructions regarding notes on sources of information and transposition.)  

I added the topical subheadings because there were a couple of rules I wanted to group together (such as No edition statement).  Looking back on the groupings, I think the individual rules can stand on their own without the topical subheadings, if we want to greater consistency.   Are there other note elements where some sort of topical grouping was warranted/helpful?

**Comment by lizoknj** [comment]
That makes sense, Michelle. Seems like a good idea to vary the approach depending on what makes most sense in a given type of note, or, for that matter, element. Some of the physical description elements are so brief, consisting essentially  of a single instruction, that it seems like overkill to use the heading "General rule".
